src/playground.gleam

```diff
@@ -246,6 +259,12 @@ fn require(
   }
 }
 
+fn read_gleam_version() -> snag.Result(String) {
+  gleam_version
+  |> simplifile.read()
+  |> file_error("Failed to read glema version at path " <> gleam_version)
+}
```

Comment:
Note that at runtime you may be able to use this as well `build/dev/javascript/gleam_version`.

Reply:
Actually nevermind, that would be what gleam version ran on the host to do the precompiling.
The compiler itself would be in `playground/wasm-compiler/package.json`